src/Entities/Keyboard.php

```diff
@@ -109,7 +109,8 @@ protected function createFromParams(): array
             $data[$keyboard_type] = $new_keyboard;
         }
 
-        return $data;
+        // If $args was empty, $data still contains `false`
+        return $data ?? [];
```

Comment:
Ah, this should be `?:` instead of `??`.

Reply:
You're right, I only tested it in PHP's interactive shell, and due to a typo it worked there! Fixed now